analyzer/src/common/TSetupData.cpp

```diff
@@ -25,38 +25,17 @@ void TSetupData::GetAllDetectors(std::vector<std::string>& detectors)const{
    }
 }
 
-int TSetupData::GetNBits(const std::string& bk)const{
+int TSetupData::GetNBits(const std::string& bk) const {
   if (bk[0] == 'D')
     return GetNBits(bk[1] - '0');
-  if (bk[0] == 'S' && 
-      bk[1] == 'I' &&
-      bk[2] == 'S' &&
-      bk[3] == '3' &&
-      bk[4] == '3' &&
-      bk[5] == '5' &&
-      bk[6] == '0')
-    return 12;
-  if (bk[0] == 'S' && 
-      bk[1] == 'I' &&
-      bk[2] == 'S' &&
-      bk[3] == '3' &&
-      bk[4] == '3' &&
-      bk[5] == '0' &&
-      bk[6] == '0')
-    return 12;
-  if (bk[0] == 'S' && 
-      bk[1] == 'I' &&
-      bk[2] == 'S' &&
-      bk[3] == '3' &&
-      bk[4] == '3' &&
-      bk[5] == '0' &&
-      bk[6] == '1')
-    return 14;
+  else if (bk[0] == 'S')
+    return GetNBits(1);
```

Comment:
Doesn't this ignore the SIS3301 board which was 14 bits? Or was that board never used?